.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: ruby
+sudo: false
+dist: bionic
+
+services:
+  - postgresql
+
+cache:
+  bundler: true
+
+addons:
+  postgresql: "9.6"
+
+before_install:
+  - gem update --system 3.0.6
+  - gem install bundler
+
+before_script:
+  - RAILS_ENV=test bundle exec rake db:create
+
+rvm:
+  - 2.5.7
+
```

Comment:
Add 2.6.5 to rvm

Reply:
Added in https://github.com/boston-library/bpldc_authority_api/pull/2/commits/3531b1c82e315f01a3182727ded16c61ee22ab6a